Userland/Libraries/LibWeb/CSS/Properties.json

```diff
@@ -18,7 +18,7 @@
   },
   "align-items": {
     "inherited": false,
-    "initial": "stretch",
+    "initial": "normal",
```

Comment:
Hooray for having multiple specs defining this, which disagree. :tada: But yes ALIGN-3 seems the better one to go with.

Reply:
Indeed! See also this comment in CSS-FLEXBOX-1 (https://www.w3.org/TR/css-flexbox-1/#alignment)
> 
> Note: While the alignment properties are defined in [CSS Box Alignment](https://www.w3.org/TR/css3-align/) [[CSS-ALIGN-3]](https://www.w3.org/TR/css-flexbox-1/#biblio-css-align-3), Flexible Box Layout reproduces the definitions of the relevant ones here so as to not create a normative dependency that may slow down advancement of the spec. These properties apply only to flex layout until [CSS Box Alignment Level 3](https://www.w3.org/TR/css3-align/) is finished and defines their effect for other layout modes. Additionally, any new values defined in the Box Alignment module will apply to Flexible Box Layout; in otherwords, the Box Alignment module, once completed, will supercede the definitions here.